pimd/pim_instance.c

```diff
@@ -51,6 +51,7 @@ static void pim_instance_terminate(struct pim_instance *pim)
 	pim_instance_mlag_terminate(pim);
 
 	pim_upstream_terminate(pim);
+	pim_upstream_timers_stop(pim);
```

Comment:
In the function pim_upstream_terminate, since rb_pim_upstream_fini for the tree is called, so the upstreams are no more accessible. So the code here to turn off the timers for the remaining upstreams won't work.

Reply:
fixed